node/src/Commands.ts

```diff
@@ -1549,3 +1550,22 @@ export function createObjectIdletime(key: string): command_request.Command {
 export function createObjectRefcount(key: string): command_request.Command {
     return createCommand(RequestType.ObjectRefCount, [key]);
 }
+
```

Comment:
Ok, it is more complicated than what I originally thought by adopting class method, but it may be benefit for some commands which have very complicated options and logic. 
For my `lolwut` command, I use a simpler approach.  What do you think?
![image](https://github.com/user-attachments/assets/019a5778-5b51-411b-824f-f63f4b55f280)

Reply:
I think we want to have the same structure across the board as it would be unclear where we draw the line of creating a class vs an interface. The PR is okay for now and I can refactor when we decide. I think based on Yury's comments the implementation of Options can be simpler.